pubspec.yaml

```diff
@@ -6,59 +6,37 @@ authors:
   - Workiva UI Platform Chapter <uip@workiva.com>
   - Workiva App Frameworks Team <appframeworks@workiva.com>
 environment:
-  sdk: '>=2.0.0-dev.0 <=2.0.0-dev.49.0'
+  sdk: '>=2.1.0 <3.0.0'
 dependencies:
-  analyzer: ">=0.30.0 <=0.31.0"
-  barback: ">=0.15.2 <=0.15.2+14"
+  # analyzer >= 0.33.0 breaks the builder, and analyzer < 0.32.4 is Dart 2 incompatible. Wdesk currently resolves to
+  # 0.30.x, so we need the lower bound
+  analyzer: ">=0.30.0 <0.33.0"
+  build: ^1.0.0
   built_redux: ^7.4.2
   built_value: ^5.4.4
   js: ^0.6.1+1
   logging: ">=0.11.3+2 <1.0.0"
   meta: ^1.1.6
   path: ^1.5.1
   react: ^4.5.0
+  source_gen: ^0.9.0
   source_span: ^1.4.1
-  transformer_utils: ^0.1.5
+  transformer_utils: ^0.2.0
   w_common: ^1.13.0
   w_flux: ^2.9.5
   platform_detect: ^1.3.4
-  quiver: ">=0.25.0 <0.29.0"
+  quiver: ">=0.25.0 <1.0.0"
 dev_dependencies:
-  build_runner: ^0.8.8
-  built_value_generator: ^5.3.0
-  coverage: ">=0.10.0 <0.12.0"
+  build_resolvers: ^0.2.0
+  build_runner: any
+  build_test: any
+  build_web_compilers: any
```

Comment:
Are these `any` ranges a good idea for pub solver times?

Reply:
They're limited by the build ^1.0.0 constraint. Also since they're dev dependencies they don't have a compounding affect on consumers. Fwiw, a pub get solves in ~3 secs.